contents/blog/new-pricing.md

```diff
@@ -0,0 +1,144 @@
+
+This will track anonymous traffic without person profiles, and start collecting person information any time an identifying action is taken (i.e. using `identify()`, `group()`, setting person properties with `$set`, etc).
+
+You can get more details on this and other use cases, like server-side events, in our [persons documentation](/docs/data/persons).
+
+## What's the downside?
+
+When sending events without person information, you cannot:
+
+- Filter on persons (e.g. an individual user)
+- Create cohorts filtered on person properties (e.g. paying vs non-paying users)
+- Target by person properties for feature flags, A/B tests, and surveys.
+- Receive initial UTM values for tracking marketing campaign performance (only the most recent).
+- View a person’s profile in the app, or query the persons table in our SQL insights.
+- Use [groups functionality](/docs/product-analytics/group-analytics) on standard events (without person profiles).
+
+If you need the above functionality for all your traffic, you don't need to change anything. Continue to send person information with your events.
+
+Person profiles are not all-or-nothing. You can use them on any traffic where you need more detailed information, and skip them when you don't need that detail. Read more about this in our [persons documentation](/docs/data/persons).
+
+## Why are we doing this?
+
+The short answer? Because we can.
+
+The slightly longer answer? Because it's the right thing to do for both our users and our business.
+
+Most of our competitors are inefficient. They employ huge [outbound sales teams](/founders/negotiate-software-better) to grow revenue. Their salaries and commissions for closing deals are passed onto their customers through higher prices.
+
+In contrast, we're 100% inbound, we grow mostly through word of mouth, and we charge based on actual usage. 
+
+We don't believe in loss-leaders, so we make a modest positive margin on each event sold, but what we charge is directly connected to what it costs us, not what we think we get away with charging.
+
+We grow our revenue through helping you grow, and onboarding you onto other tools, like session replay, feature flags, and surveys as well.
+
+We think this is better than trying to squeeze you for every cent you have. You have a great experience and we'll enjoy better retention and word of mouth for doing the right thing.
+
+> ## FAQ
+>
+> #### Will my bill go up if I continue to use person profiles?
+>
+> No. Events with person profiles enabled cost the same as our old pricing. You'll see a new line item on your invoice, but what you pay won't change if you send person profiles on every event. 
+>
+> #### Why aren't you making the events with person profiles cheaper?
+>
+> Because events with person profiles cost a lot more for us to ingest and query. Not sending person information for certain events, however, saves us money, so we're passing that on.
```

Comment:
```suggestion
> Because events with person profiles cost a lot more for us to ingest and query. Not sending person data for certain events, however, saves us money, so we're passing that on.
```